examples/produce_part_sweep/README.md

```
@@ -0,0 +1,35 @@
+```
+
+
+## Parts
+
+### clock
+<table><tr>
+<td valign=top><a href="clock.sweep"><img src="././clock.svg" style="width: auto; height: auto; max-width: 200px; max-height: 200px;"></a></td>
+<td valign=top>Sweep a clock sketch to create a 3D clock.</td>
+</tr></table>
+
+### dxf
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)